src/theme/NavbarItem/NavbarNavLink.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Link from '@docusaurus/Link';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import isInternalUrl from '@docusaurus/isInternalUrl';
+import {isRegexpStringMatch} from '@docusaurus/theme-common';
+import IconExternalLink from '@theme/Icon/ExternalLink';
+export default function NavbarNavLink({
+  activeBasePath,
+  activeBaseRegex,
+  to,
+  href,
+  label,
+  html,
+  isDropdownLink,
+  prependBaseUrlToHref,
+  // Start Playwright hack
+  skipExternalLinkCheck,
+  // End Playwright hack
+  ...props
+}) {
+  // TODO all this seems hacky
+  // {to: 'version'} should probably be forbidden, in favor of {to: '/version'}
+  const toUrl = useBaseUrl(to);
+  const activeBaseUrl = useBaseUrl(activeBasePath);
+  const normalizedHref = useBaseUrl(href, {forcePrependBaseUrl: true});
+  // Start Playwright hack
+  const isExternalLink = label && href && !skipExternalLinkCheck && !isInternalUrl(href);
```

Comment:
and this was modified. The rest was generated by `npm run swizzle -- @docusaurus/theme-classic NavbarItem/NavbarNavLink`